PWGCF/FemtoWorld/Tasks/femtoWorldEfficiencyTask.cxx

```diff
@@ -417,6 +426,16 @@ struct femtoWorldEficiencyTask {
             // PID only
             registryPri.fill(HIST("plus/AllPri"), track.pt(), track.eta());
             registryPri.fill(HIST("plus/AllPriPt"), mcParticle.pt());
+            // generated primary, no cuts, no nsigma
+            if (mcParticle.pdgCode() == 211) {
+              registryPri.fill(HIST("plus/GenPriPi"), mcParticle.pt());
+            }
+            if (mcParticle.pdgCode() == 321) {
+              registryPri.fill(HIST("plus/GenPriKa"), mcParticle.pt());
+            }
+            if (mcParticle.pdgCode() == 2212) {
+              registryPri.fill(HIST("plus/GenPriPr"), mcParticle.pt());
+            }
```

Comment:
For the next iteration, please, consider using
```
if (...) {
  ...
} else if (...) {
  ...
} else if (...) {
  ...
}
```
it will be more efficient